docs/download.md

```diff
@@ -41,6 +41,27 @@ for your hardware's minimum driver version.
 *For Cloudera and EMR support, please refer to the
 [Distributions](./FAQ.md#which-distributions-are-supported) section of the FAQ.
 
+### Download v22.10.0
+* Download the [RAPIDS
+  Accelerator for Apache Spark 22.10.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/22.10.0/rapids-4-spark_2.12-22.10.0.jar)
+
+This package is built against CUDA 11.5 and all CUDA 11.x versions are supported through [CUDA forward
+compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/index.html). It is tested
+on V100, T4, A2, A10, A30 and A100 GPUs with CUDA 11.0-11.5.  For those using other types of GPUs which
+do not have CUDA forward compatibility (for example, GeForce), CUDA 11.5 or later is required. Users will
+need to ensure the minimum driver (450.80.02) and CUDA toolkit are installed on each Spark node.
```

Comment:
Nit: Not sure about "CUDA toolkit are installed on each Spark node."
If we don't use GDS or UCX, I think it may not need cuda toolkit.
I'll approve this PR and we can check this later.

Reply:
yes, on dataproc we don't have a GPU on driver node. should we also draft a pr in dev 2210 branch?